docs/docs/guides/labs/observability-update/asset-catalog.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Custom asset catalog dashboards
+description: Flexibly build dashboards in the asset catalog scoped by tags, teams, owners, or asset groups in order to enable everyone on your team to focus on the assets that matter most to them.
+sidebar_position: 500
+---
+
+In the asset catalog, you can now build dashboards scoped by tags, teams, owners, or asset groups, enabling you to focus on the assets that matter most.
+
+To build a dashboard, create saved selections from Catalog search. For more information, see the [asset selection syntax documentation](/guides/build/assets/asset-selection-syntax).
```

Comment:
```suggestion
To build a dashboard, create saved selections from Catalog search. For more information, see the [asset selection syntax documentation](/guides/build/assets/asset-selection-syntax/reference).
```

I think the info users would be looking for from here is the specifics of the syntax, not the other places you can use asset selections.

Reply:
There are new functions in the syntax that are Observe specific (at least `status` which lets you filter on asset health status; @salazarm are there any others?).

It probably belongs in the filters section (or something equivalent) of the selection syntax reference page:
> Filter: Asset health status
> Syntax: `status: "DEGRADED"`
> Description: Select assets with a given asset health status ([link to docs]). Options are `"HEALTHY", "WARNING", "DEGRADED", "UNKNOWN" or "NOT_APPLICABLE."
> Supported views: Dagster+ only (requires Observability Early Access [link])